tools/repo_setup.py

```diff
@@ -369,6 +381,55 @@ def update_repo(self, submodule):
         else:
             print_error("Error loading submodule info")
 
+    def get_archives(self, archive):
```

Comment:
Pra função não ficar gigante se/quando tivermos outras formas de download de arquivo... Aí basta implementar outras funções

```suggestion
    def get_archives_az_artifacts(self, archive):
```